index.html

```diff
@@ -28,6 +34,7 @@ <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
             <h2 class="lets-connect">LET'S CONNECT</h2>
         </a>
 <br>
+<div class="social-mv">
 <ul class="connect-via">
     <li><img  src="assets//images/twitter.png" alt="twitter"> </li>
```

Comment:
- [ ] OPTIONAL: I recommend wrapping social icons with <a> tag, because they are supposed to be links and centre them horizontally.  😉